sqldef.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type Options struct {
 	DesiredFile string
```

Comment:
```suggestion
```

In case it wasn't clear, at https://github.com/k0kubun/sqldef/pull/315#discussion_r1031861301, I suggested to remove this field. The caller should be able to read the `DesiredFile`, and pass that as `DesiredDDLs`. Having both in `Options` seems redundant and confusing. It's nicer to make unexpected situations simply not compilable instead of making it a runtime error.

Reply:
I made a misunderstanding. You are right.

Reply:
Each `*sql*def.go` has become somewhat redundant. What do you think?